website/src/components/index-page/hero.tsx

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link"
+import { CodeA, CodeB, CodeC } from "../code-blocks"
+import { GraphQLLogo } from "@/icons"
+import { clsx } from "clsx"
+
+export function Hero() {
+  return (
+    <div className="hero dark">
+      <div className="container conf-block">
+        <section
+          className={clsx(
+            "flex-wrap gap-14 justify-center items-center flex [&_pre]:inline-block max-sm:flex-col",
+            "[&_h3]:text-white [&_h3]:text-2xl max-lg:[&_h3]:text-center",
+            "[&_pre]:!bg-transparent [&_pre]:ring-0 [&_pre_span]:text-[--shiki-dark]",
+            '[&_h3]:font-extralight'
+          )}
+        >
+          <div className="named-logo max-md:grow max-xl:w-full flex flex-col items-center gap-2">
+            <GraphQLLogo className="size-24" />
+            <h1 className="text-primary text-3xl">GraphQL</h1>
+          </div>
+
+          <div className="marketing-col">
+            <h3>Describe your data</h3>
+            <CodeA />
+          </div>
+
+          <div className="marketing-col">
+            <h3>Ask for what you want</h3>
+            <CodeB />
+          </div>
+
+          <div className="marketing-col">
+            <h3>Get predictable results</h3>
+            <CodeC />
+          </div>
```

Comment:
The animations here are rather slow, and given they are first text to render this could effect page ranking score and user experience due to first meaningfull paint. Maybe we just remove the animations at the top of the page. 

https://web.dev/articles/vitals

<img width="1538" alt="Screenshot 2024-03-19 at 10 58 26" src="https://github.com/graphql/graphql.github.io/assets/35999252/3c5033dd-3e3d-4b46-8276-9e27730796ed">